viz.yaml

```diff
@@ -41,5 +44,12 @@ process:
     depends: 
       waterUse: waterUse-data
 visualize:
+  -
+    id: wateruse-treemap
+    location: cache/treemaps/
+    visualizer: treemap
+    scripts: scripts/visualize/treemap.R
+    depends:
+      waterUse_clean: calc-waterData
```

Comment:
It is a layer of indirection, the user of a dependency can call it one thing, but the thing injecting the dependency (via yaml) can call it something else.  This will help us to make generalized components (fetchers, processors, etc) without our hands being tied.